app/build.gradle.kts

```diff
@@ -144,24 +144,23 @@ dependencies {
     kotlinDependencies()
     coroutineDependencies()
 
+    // Koin DI
+    koinDependencies()
+
     // AndroidX
     composeDependencies()
     accompanistDependencies()
     appCompatDependencies()
     activityDependencies()
-    fragmentDependencies()
```

Comment:
@BottleRocket-Colin It looks like a few additional fragment related things can be removed:

* `FRAGMENT_VERSION`
* `ANDROIDX_FRAGMENT_TEST`
* `fragmentTestingDependencies` (definition and usage in app `build.gradle.kts`)

I'll remove this in a review commit.